sdk/communication/azure-communication-common/README.md

```diff
@@ -20,7 +20,7 @@ This package contains common code for Azure Communication Service libraries.
   APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
 
 ### Versions available
-The current version of this library is **1.0.0-beta.5**.
+The current version of this library is **1.0.0-beta.7**.
```

Comment:
```suggestion
The current version of this library is **1.0.0-beta.9**.
```

Reply:
The README should point to the latest released version.

Reply:
After Monday, beta.9 is the latest